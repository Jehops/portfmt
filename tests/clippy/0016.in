license=	GPLv2
.include <bsd.port.mk>
