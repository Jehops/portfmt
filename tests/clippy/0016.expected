
# Unknown variables
# WARNING:
# Portclippy did not recognize the following variables.
# They could be local variables only, misspellings of
# framework variables, or Portclippy needs to be made aware
# of them.  Please double check them.
#
# Prefix them with an _ to tell Portclippy to ignore them.
# This is also an important signal for other contributors
# who are working on your port.  It removes any doubt of
# whether they are framework variables or not and whether
# they are safe to remove/rename or not.
license     did you mean LICENSE ?
